tests/containers/bats/aardvark.pm

```diff
@@ -57,7 +57,7 @@ sub run {
     select_serial_terminal;
 
     # Install tests dependencies
-    my @pkgs = qw(aardvark-dns firewalld iproute2 iptables jq netavark podman);
```

Comment:
What about SLES 15-SP3 and SLEM where we still run those tests?

Reply:
We don't run these tests on SLEM.  Also, aardvark is only run on Tumbleweed.

Reply:
Ah ok, yes then this is fine.